FunMotifs.pm

```diff
@@ -65,10 +67,9 @@ sub new {
   my $class = shift;
 
   my $self = $class->SUPER::new(@_);
-  if(scalar(@{$self->params}) < 3){
-    warn('Insufficient input parameters found for FunMotifs plugin');
-    return $self;
-  }
+  
+  die('Insufficient input parameters found for FunMotifs plugin') if(scalar(@{$self->params}) < 2);
```

Comment:
For consistency with die messages from other plugins  can you please add a newline to the die message? This will prevent the die method from printing the line number.